src/ontology/mondo-edit.obo

```diff
@@ -395960,18 +395957,17 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 
 [Term]
 id: MONDO:0021016
-name: channelopathy
+name: obsolete channelopathy
 def: "A disease caused by disturbed function of ion channel subunits or the proteins that regulate them." [Wikipedia:Channelopathy]
```

Comment:
@sabrinatoro the definitions should be prepended with OBSOLETE, see: https://mondo.readthedocs.io/en/latest/editors-guide/merging-and-obsoleting/#obsolete-a-class-manually

(this is a recent change)

Reply:
I added this, no action required from you. just FYI :)